pkg/roachprod/promhelperclient/client.go

```diff
@@ -237,20 +178,53 @@ func ProviderReachability(provider string, cloudEnvironment CloudEnvironment) Re
 	return providerReachability[Default]
 }
 
-// CCParams are the params for the cluster configs
-type CCParams struct {
-	Targets []string          `yaml:"targets"`
-	Labels  map[string]string `yaml:"labels"`
+// getUrl returns the URL for the prometheus helper service for a given cluster
+func (c *PromClient) getUrl(clusterName string) string {
+	return fmt.Sprintf("%s/%s/%s/%s", c.promUrl, resourceVersion, resourceName, clusterName)
 }
 
-// NodeInfo contains the target and labels for the node
-type NodeInfo struct {
-	Target       string            // Name of the node
-	CustomLabels map[string]string // Custom labels to be added to the cluster config
+// makeRequest makes the http request and returns an error with the body if an error occurs
+func (c *PromClient) makeRequest(request *http.Request, logPrefix string) error {
+	response, err := c.client.Do(request)
+	if err != nil {
+		return errors.Wrapf(err, "%s: failed on url: %s", logPrefix, request.URL)
```

Comment:
nit: it probably makes more sense for the caller of `makeRequest` to do the error wrapping - if they want